### PR TITLE
chore: remove unused message strings

### PR DESCRIPTION
**Pull request by zainab-amir**
### Description

- Removed unused message strings.
- Removed the file src/common-components/SwitchContent.jsx as it is no longer being used.

#### How Has This Been Tested?

- Unit tests are passing.
- Locally running the authn MFE

#### Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Sandbox, if applicable.
* [ ] Is there adequate test coverage for your changes?

#### Post-merge Checklist

* [ ] Deploy the changes to prod after verifying on stage or ask **@openedx/vanguards** to do it. 
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

File: src/forgot-password/messages.js [hunk 1]
```diff
@@ -51,11 +51,6 @@ const messages = defineMessages({
     defaultMessage: 'Enter your email',
     description: 'Error message that appears when user tries to submit empty email field',
   },
-  'forgot.password.invalid.email.message': {
```
Comment (syedsajjadkazmii) [inline src/forgot-password/messages.js:54]: I see its usage in `ForgotPasswordPage.jsx`. why are we removing it?

Comment (zainab-amir) [reply src/forgot-password/messages.js:54]: Can you share the code reference?

File: src/common-components/RedirectLogistration.jsx [hunk 2]
```diff
@@ -26,6 +26,7 @@ function RedirectLogistration(props) {
 
     // Redirect to Progressive Profiling after successful registration
     if (redirectToWelcomePage) {
+      // TODO: Do we still need this cookie?
```
Comment (syedsajjadkazmii) [inline src/common-components/RedirectLogistration.jsx:29]: were you able to get its answer?

File: src/forgot-password/messages.js (hunk 1 above)
Comment (syedsajjadkazmii) [reply src/forgot-password/messages.js:54]: sorry, it was confusion at my side.

File: src/common-components/RedirectLogistration.jsx (hunk 2 above)
Comment (zainab-amir) [reply src/common-components/RedirectLogistration.jsx:29]: No, I have added it for future reference.

**Review by syedsajjadkazmii — APPROVED** [review]

**Pull request merged**